MIGRATION.md

```diff
@@ -20,7 +20,14 @@ content like using `raw_result=True` in `python-cloudant`.
 1. Replay adapters are replaced by the [automatic retries](https://github.
    com/IBM/ibm-cloud-sdk-common/#automatic-retries) feature for failed requests.
 1. Error handling is not transferable from `python-cloudant` to `cloudant-python-sdk`. For more information go to the [Error handling section](https://cloud.ibm.com/apidocs/cloudant?code=python#error-handling) in our API docs.
-1. Custom HTTP client configurations in `python-cloudant` are not transferable to `python-java-sdk`. For more information go to the [Configuring the HTTP client section(https://githubcom/IBM/ibm-cloud-sdk-common/#configuring-the-http-client) in the IBM Cloud SDK Common README.
+1. Custom HTTP client configurations in `python-cloudant` are not transferable to 
+   `cloudant-python-sdk`. For more information go to the [Configuring the HTTP client section
+   (https://github.com/IBM/ibm-cloud-sdk-common/#configuring-the-http-client) in the IBM Cloud 
+   SDK Common README.
+1. Authentication errors turn out at the time of instantiation of a client, while errors
```

Comment:
Would you plan to update this chapter too, like you did with java and nodejs?

Reply:
yes, I just need to figure out what is the authentication error message first

Reply:
Done in https://github.com/cloudant/python-cloudant/pull/512/commits/15a9b609719fb907efa72214a2ab9d050e3c37f5